backend/src/chats/message/entities/message.entity.ts

```diff
@@ -0,0 +1,35 @@
+import {
+	Column,
+	CreateDateColumn,
+	Entity,
+	JoinColumn,
+	ManyToOne,
+	OneToOne,
+	PrimaryGeneratedColumn,
+	UpdateDateColumn,
+} from 'typeorm';
+import { Chat } from '../../chat/entities/chat.entity';
+import { User } from '../../../users/user/entities/user.entity';
+
+@Entity('messages')
+export class Message {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@CreateDateColumn()
+	created_at: Date;
+
+	@UpdateDateColumn()
+	updated_at: Date;
+
+	@OneToOne((type) => User, (user) => user.id)
+	@JoinColumn({ name: 'sender_id' })
+	sender_id: User;
+
+	@ManyToOne((type) => Chat, (chat) => chat.id)
+	@JoinColumn({ name: 'chat_id' })
+	chat_id: Chat;
+
+	@Column({ nullable: false })
```

Comment:
The DTO states that the content is `@Optional()`, but `nullable: false` in the entity so sending an empty message throws a 400 error.